scripts/train_lora.sh

```diff
@@ -0,0 +1,30 @@
+deepspeed --num_gpus 6 fastchat/train/train_lora.py \
```

Comment:
let's not add the `--num_gpus` flag since we don't know the number of gpus from the user. maybe they only have one gpu